appcues/src/main/java/com/appcues/ui/AppcuesViewModel.kt

```diff
@@ -94,7 +98,10 @@ internal class AppcuesViewModel(
             val stepIndexInContainer = stepIndexLookup[flatStepIndex]
             // if both are valid ids we return Rendering else null
             if (containerId != null && stepIndexInContainer != null) {
-                Rendering(stepContainers[containerId], stepIndexInContainer)
+                // set the current experience id for later usage on refresh preview
+                currentExperienceId = id
```

Comment:
quick initial scan - maybe we should only set this value if we know it is a preview (`published.not()`) just in case?

Reply:
check it again to see if its easier to shake and also see if setting the experienceId in Rendering looks nicer.

Reply:
@andretortolano this is feeling pretty good to me - It's still a little bit harder on my Pixel 2 XL than I'm used to on the iphone, have to do sort of a hard 90 degree shake down, but it works. And I'd really like to see what others thing since this may just be a device specific thing - this is certainly an older phone.

Reply:
okay!